todo_app/data/trello_items.py

```diff
@@ -4,13 +4,38 @@
 
 import os
 
-boardid = os.environ.get('BOARDID')
-trello_key = os.environ.get('TRELLO_KEY')
-trello_token = os.environ.get('TRELLO_TOKEN')
 
-listtodo = os.environ.get('LISTTODO')
-listdoing = os.environ.get('LISTDOING')
-listdone = os.environ.get('LISTDONE')
+#trello_key = os.environ.get('TRELLO_KEY')
+#trello_token = os.environ.get('TRELLO_TOKEN')
+
+#boardid = os.environ.get('BOARDID')
+#listtodo = os.environ.get('LISTTODO')
+#listdoing = os.environ.get('LISTDOING')
+#listdone = os.environ.get('LISTDONE')
+
+def get_trello_key():
```

Comment:
This is good, but don't forget to make sure you're calling the function everywhere you use it, rather than passing the function itself - there's still a few examples of that issue